util/app.py

```diff
@@ -36,82 +35,23 @@ def run(self):
 
 
     def _run(self):
+        import ui.gui as gui
+        import ui.cli as cli
```

Comment:
Why not import them inside the corresponding `case` below?

Reply:
`cli.console_app` is called in two cases, would be redundant to put an import inside both of them. The reason why these aren't at the top level is circular imports.